Makefile

```
@@ -86,12 +86,21 @@ $(AMD64_BINS): GOARCH=amd64
 $(ARM64_BINS): GOARCH=arm64
 
 .PHONY: build
-build: $(DRIVER_BINS)
+build: vet fmt $(DRIVER_BINS)
 
-$(DRIVER_BINS): vet fmt
+$(DRIVER_BINS):
 	@CGO_ENABLED=0 GOTOOLCHAIN=local GOOS=linux GOARCH=$(GOARCH) \
 		go build -ldflags "${LDFLAGS}" -o $@ ./cmd
```

I do not love this whole `./` here. I am thinking we either make the path absolute, or just leave `cmd`.
Parentheses instead of curly braces.
```suggestion
		go build -ldflags "$(LDFLAGS)" -o $@ ./cmd
```

---

Changing `./cmd` to `cmd` results in go build failure so we'll leave as `./cmd`

go: cannot write multiple packages to non-directory bin/brickstor-csi-driver_linux_amd64_v1.0.0